src/button-group.test.basics.ts

```diff
@@ -90,8 +90,8 @@ it('sets `privateVariant` on its buttons', async () => {
 
   const buttons = host.querySelectorAll('glide-core-button-group-button');
 
-  expect(buttons[0].privateVariant).to.equal('icon-only');
-  expect(buttons[1].privateVariant).to.equal('icon-only');
+  expect(buttons[0]?.privateVariant).to.equal('icon-only');
```

Comment:
How would you feel about doing `assert` for all these instead?

```ts
assert(buttons[0]);
assert(buttons[1]);
expect(buttons[0].privateVariant).to.equal('icon-only');
expect(buttons[1].privateVariant).to.equal('icon-only');
```

That way, if it fails because one of the elements is undefined, it fails on that specific line, making it more clear what the failure is

Reply:
I like `assert` too. It would just be verbose if we used it all over the place.